### DISCUSSION TITLE
Contribution Graph

### DISCUSSION THREAD

**Discussion by AhmadSharafat** [community/community#140183] [Mobile] [answered]
Hey as the GitHub announced the GitHub graph will be available on Android mobile apps but I didn't receive it yet I update the app to latest version and also am in the GitHub beta Tester program but I didn't receive it yet what's the way to get it or when it's available in Pakistan origin

**Comment by stevepopovich** [comment]
Hi @AhmadSharafat can you screenshot what you see when you long press the GitHub app? It should be available to you.

**Reply by AhmadSharafat** [reply]
![Screenshot_20240930-195412.jpg](https://github.com/user-attachments/assets/c9890624-f416-44df-9563-976ec1ef97b5)

Hey @stevepopovich this appears when I long press the app and when I long press on home screen nothing appears!

**Reply by stevepopovich** [reply]
Hmm what version of Android are you using? Can you use other app widgets? Your operating system may have a different way of adding widgets from apps.

**Reply by AhmadSharafat** [reply]
![Screenshot_20240930-205357.jpg](https://github.com/user-attachments/assets/8bfe80c6-1510-46ac-a6cb-825b40655530)

These are the details of mobile kindly share the way to add widgets if possible or share the other solutions I don't understand what's the problem Steve!

**Reply by AhmadSharafat** [reply]
And when I long press other apps who have widgets feature or create shortcuts I can access them but not happening with GitHub Mobile app

**Reply by AhmadSharafat** [reply]
![Screenshot_20240930-210212.jpg](https://github.com/user-attachments/assets/bdcf9111-fc0d-4e93-8a27-79d53e0486f1)

Here take a look of updated features on Play store there is no update about contribution graph feature in these updates

**Reply by stevepopovich** [reply]
That is certainly the latest beta release notes. You are on the correct version. I would say there is something going on with your specific operating system, since this is a slightly less common operating system. Could you switch to the non-beta version of the app?

**Reply by AhmadSharafat** [reply]
Ok I try the non beta App but I can use the widgets of other apps on my phone only GitHub showing problem

**Reply by AhmadSharafat** [reply]
Now I try the non beta app but still the same the contribution graph widget not appearing am trying this from last 2 days 
Any solutions Steve?

**Reply by stevepopovich** [reply]
The app is installed from the Google Play store, correct?

**Reply by AhmadSharafat** [reply]
Yeah Steve from Google Play Store

**Reply by stevepopovich** [reply]
I fear this has to do with you running Dolphin OS and not a more "stock" version of Android. I will converse with the team about this a bit more

**Reply by AhmadSharafat** [reply]
Hey Steve I find a way to add feature but after adding the feature on my mobile nothing appears just a black screen and with a dot on corner and when I click on GitHub app opens

**Reply by AhmadSharafat** [reply]
![IMG-20241001-WA0000.jpg](https://github.com/user-attachments/assets/0d73b587-b621-4766-8359-e0fa9f4753e9)

Here take a look widget look like this

**Reply by stevepopovich** [reply]
Have you installed a third party launcher? [It seems like Infinix devices need this for widgets to work.](https://www.infinix.club/forum/195/977285)

**Reply by AhmadSharafat** [reply]
No just long press my Apps Screen and below widgets option appear which contain all widgets of all the apps and i go to github widget and click on add to desktop then this appears now today i make three contribution to my repo this black Dot becomes green with that not all the graph appears

**Reply by stevepopovich** [reply]
Ok, not sure I am fully understanding here. You are saying the widget works for a few days and then stops working? From your screenshot, it looks like your widget does not work at all

**Reply by AhmadSharafat** [reply]
Yeah! Steve Widget didn't work at all but the other thing is from last two days am not able to add widget So today i find way to add it but it didn't work  and i share the screenshot with you that widget is not working and the graph is not appearing fully  Kindly look into it the main problem now Widget is not Working As you see in Screeshot

**Reply by stevepopovich** [reply]
Did you try this link? https://www.infinix.club/forum/195/977285

I don't see any other widgets working so you may need a third party launcher

**Reply by AhmadSharafat** [reply]
![Screenshot_20241001-212534.jpg](https://github.com/user-attachments/assets/fca915ce-cb90-4000-9918-68306c14a068)

Steve take a look My Calender wideget is working as you see in my screenshot and with that all the widgets from other apps is working but problem is appearing from GitHub widget as I share the screenshot in the previous messages

**Comment by HoussemZorgui** [comment]
Hi,

The contribution graph for Android is being rolled out gradually, so it might not be available in all regions, including Pakistan, yet. Since you're in the beta program, just keep your app updated, and it should appear soon.

**Reply by stevepopovich** [reply]
It should be available to all regions at this time.

**Reply by AhmadSharafat** [reply]
As you said it's available I try it same as before with that I also reinstall the app but same problem I am exhausted Steve

**Reply by HoussemZorgui** [reply]
@stevepopovich I hope it will be available in all regions soon.

**Reply by HoussemZorgui** [reply]
@AhmadSharafat 
Thank you for confirming that it's available. I tried it again as I did before, and I also reinstalled the app, but I'm still facing the same issue. I'm really exhausted, Steve.

**Comment by stevepopovich** [comment]
We will file this as a bug and investigate this in time. Thank you!

**Reply by AhmadSharafat** [reply, marked as answer]
Looking forward to the best side Steven!
